app/matplotlib.ipynb

```diff
@@ -47,21 +47,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
-    "collapsed": true,
-    "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
-    }
+    "collapsed": true
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "# write your code here\n",
+    "plt.plot(years, population)\n",
+    "\n",
+    "plt.title(\"Population over years\")\n",
+    "plt.xlabel(\"Years\")\n",
+    "plt.ylabel(\"Population\")\n",
+    "\n",
+    "plt. show()"
```

Comment:
There is an extra space in the `plt.show()` function call. It should be `plt.show()` without the space.